DOC-1528/ttp-step-1-overview-page-indexmdx (#541)
swan-io/docs: docs/preview/in-person/cards/index.mdx

title: In-person cards
---

# In-person cards

For many businesses, accepting card payments in person is not just a convenience; it's a must-have.
From brick-and-mortar shops to health professionals, home care service providers, taxi drivers, and contractors, many professionals need to accept card payments on the go or on their own premises.

With Swan's API, you can let merchants turn their smartphones into a card terminal and accept payments through a simple terminal app in just a few taps.

:::info Merchants only
This section only applies to **merchants** accepting card payments.
Visit the [card payments section](../../../topics/payments/cards/index.mdx) for information about payments made with cards *from* a Swan account.
:::

## Merchant card payments {#merchant-card-payments}

Merchant card payments include **one or more transactions**.
For example, one payment might include a capture transaction and a refund transaction, for a total of two transactions.

These transactions are grouped together in the **merchant payment object**.
Swan recommends using the merchant payment object in your integration to help merchants match orders, invoices, and sales to the correct object.

> This provides better traceability and can help reconcile issues more efficiently.

Reviewer: ```suggestion
This makes it easier to track payments and helps reconcile issues more efficiently.
```

## Card transaction types {#transaction-types}

There are **four transaction types** for in-person merchant card payments.

:::info Statuses specific to in-person card payments
These statuses are specific to in-person card payments.
Other payment methods might behave differently.
:::

Authorizations aren't included in a transaction type.
Instead, they're represented at the payment level, visible through the payment object status `Authorized` and payment object balance `totalAuthorized`.

| Transaction type | Explanation | Possible statuses |
| --- | --- | --- |
| `CardInCredit` | Captured payment that will eventually get settled via a separate Credit Transfer. | `Upcoming`, `Settled` |
| `CardInCreditReversal` | Refunded payment that will eventually get settled via a separate Credit Transfer. | `Upcoming`, `Settled` |
| `CardInCreditChargeback` | Disputed payment that will eventually get settled via a separate Credit Transfer. | `Upcoming`, `Settled` |

> | `CardInCreditChargebackReversal` | Disputed payment for which the dispute is overturned, that will eventually get settled via a separate Credit Transfer. | `Upcoming`, `Settled` |

Reviewer: ```suggestion
| `CardInCreditChargebackReversal` | Disputed payment for which the dispute is overturned, that will eventually get settled through a separate Credit Transfer. | `Upcoming`, `Settled` |
```

### Settled status {#settled-status}

When the `Settled` status is applied to these transactions, it means that a Credit Transfer has been `Booked` on the account to settle the merchant.
The individual card transactions don't impact the balance. The `Settled` status is final and doesn't affect the balance. Instead, the Credit Transfer impacts the `Booked` balance directly.

Every `Settled` transaction includes a `settlementTransactionId` field to identify the Credit Transfer impacting the balance.
Additionally, each Credit Transfer received to settle in-person card transactions includes the list of settled transactions.

> ## Payment object statuses {#payment-statuses}

Reviewer: This could benefit from a mermaid diagram showing the lifecycle. [Here](https://deploy-preview-541--incomparable-tiramisu-91a96a.netlify.app/topics/merchants/online/sdd/#payments-statuses)'s a similar one for SDDs.

Author: [DOC-1577: 📘 Add mermaid diagram for in-person card payment object statuses](https://linear.app/swan/issue/DOC-1577/add-mermaid-diagram-for-in-person-card-payment-object-statuses)

The merchant payment object has **distinct statuses** to follow a payment's lifecycle.

| Payment object status | Explanation |
| --- | --- |
| `Initiated` | The payment has been created on the terminal app and is ready to be taken. |
| `Authorized` | Authorization for the payment was received, but the funds aren't captured yet. The funds are guaranteed for the merchant. |
| `Captured` | Authorized payments for which the funds have moved out of the customer's account. This can be a final status if the customer doesn't dispute the payment or request a refund. |
| `Rejected` | Authorization denied by the issuer or by Swan. |
| `Canceled` | Card payment was canceled, authorization voided, and the funds can't be captured. |
| `Refunded` | Payments reversed by the merchant for some or all of the amount. |
| `Disputed` | Customer disputed the payment with their bank for some or all of the amount. |

## Payment object balances {#payment-balances}

The merchant payment object has **distinct balances** to follow a payment's lifecycle.
Use these balances to understand what actions have been done on a payment, and what actions can be taken (such as canceling or refunding a payment).

> | Payment object balance | Explanation |

Reviewer: This and the [cards](https://deploy-preview-541--incomparable-tiramisu-91a96a.netlify.app/topics/merchants/online/cards/#payments-balances) "Payment object balances" can be a partial as they're the same. It'll help if you need to make an update later.

Author: The online cards page and the in-person page have identical payment object balance rows. The only difference is the column header ("Card payment object balance" vs "Payment object balance"). Extracting this into a partial is a good DRY improvement.

However, this is a structural refactor that touches multiple files and the online cards page too. I'd recommend a separate small PR for this rather than bundling it here.

Author: [DOC-1578: 📘 Extract merchant payment object balances table into a shared partial](https://linear.app/swan/issue/DOC-1578/extract-merchant-payment-object-balances-table-into-a-shared-partial)

| `totalCanceled` | The total amount canceled for this merchant payment. |
| `availableToCapture` | The amount available for merchants to capture if the capture isn't automatic. |
| `totalCaptured` | The amount captured by the merchant. When the capture for card payments is automatic, `totalCaptured` is always equal to `totalAuthorized`. |
| `availableToRefund` | The amount the merchant can refund. Use this balance to know if a payment is eligible for a refund. |
| `totalRefunded` | The total amount refunded for this merchant payment. |
| `totalDisputed` | The total amount of this merchant payment disputed by the customer. |

## Rolling reserve {#rolling-reserve}

In-person card payments are **not subject to a rolling reserve** by default.
This could change as merchants start processing if Swan's monitoring systems detect suspicious patterns and activities.

## Payment amount limit {#payment-limit}

In-person card payments are **not subject to a payment amount limit** by default.
This could change as merchants start processing if Swan's monitoring systems detect suspicious patterns and activities.

## Settlement process {#settlement}

> For in-person card payments, the settlement comes in the form of a Credit Transfer.

Reviewer: ```suggestion
For in-person card payments, the settlement is a credit transfer.
```

1. When the merchant payment is created, underlying transactions are created as `Upcoming`.

> 1. When the settlement arrives in the form of a Credit Transfer, the transactions are updated to `Settled`.

Reviewer: ```suggestion
1. When the settlement arrives as a Credit Transfer, the transactions are updated to `Settled`.
```

> 1. The card transactions themselves don't affect any of the account balances. However, the Credit Transfer is immediately `Booked`, affecting the balance, making the corresponding funds available.

Reviewer: Prioritising what happens in the steps.
